cli list blocks usability improvements (#403)
grafana/tempo: docs/tempo/website/cli/_index.md

- `tenant-id` The tenant ID.  Use `single-tenant` for single tenant setups.

Options:
- `--include-compacted` Include blocks that have been compacted. Default behavior is to display only active blocks.

> - `--load-index` Also load the block indexes and perform integrity checks for duplicates. **Note:** can be intense.

Reviewer: ```suggestion
- `--load-index` Also load the block indexes and perform integrity checks for duplicates. **Note:** This can be a resource intensive process.
```

> - `Window` The time window considered for compaction purposes.

Reviewer: ```suggestion
- `Window` The window of time that was considered for compaction purposes.
```

- `Start` The earliest timestamp stored in the block.
- `End` The latest timestamp stored in the block.
- `Age` The age of the block.

> - `Duration` Time duration between start and end.

Reviewer: ```suggestion
- `Duration`Duration between the start and end time.
```

- `Idx` Number of records stored in the index (present when --load-index is specified).

> - `Dupe` Number of duplicate entries in the index. Should be zero.  (present when --load-index is specified).

Reviewer: ```suggestion
- `Dupe` Number of duplicate entries in the index (present when --load-index is specified). Should be zero. 
```
